contrib/action_recognition/r2p1d/README.md

```diff
@@ -10,12 +10,12 @@ This project provides utility scripts to fine-tune the model and examples notebo
 | --- | --- |
 | [00_webcam](00_webcam.ipynb) | A real-time inference example on Webcam stream |
 | [01_training_introduction](01_training_introduction.ipynb) | An example of training R(2+1)D model on HMDB-51 dataset |
-| [02_video_transformation](02_video_transformation.ipynb) | Examples of video transformations | 
+| [02_video_transformation](02_video_transformation.ipynb) | Examples of video transformations |
 
 Specifically, we use the model pre-trained on 65 million social media videos (IG) presented in "[Large-scale weakly-supervised pre-training for video action recognition (2019)](https://arxiv.org/abs/1905.00561)" paper.
 
 *Note: The official pretrained model weights can be found from [https://github.com/facebookresearch/vmz](https://github.com/facebookresearch/vmz) which are based on caffe2.
-In this repository, we use PyTorch-converted weights from [https://github.com/moabitcoin/ig65m-pytorch](https://github.com/moabitcoin/ig65m-pytorch).*
+In this repository, we use code and the PyTorch-converted weights from [https://github.com/moabitcoin/ig65m-pytorch](https://github.com/moabitcoin/ig65m-pytorch).*
```

Comment:
The original statement is correct. We use the model weights only, not the codes. All the codes we wrote except some video transformations which are from Torch official.

The model weights, we get from torchHub by using torch api as instructed in the github we linked (the author's github how uploaded the weight to torchHub).
```
model = torch.hub.load(
    TORCH_R2PLUS1D, model_name, num_classes=MODELS[model_name], pretrained=True
)
```

Reply:
Thanks for the clarification. I'll have a few related questions to this once we meet.